docs/DeveloperGuide.md

```diff
@@ -517,31 +565,47 @@ The NegateCommand class also extends from the abstract `Command` class in order
 By implementing the abstract `NegateCommand` class, any future implementation of commands with similar functionality as unremind, unprioritize and undone will simply extend from the `NegateCommand` class, thereby enforcing the **Open-Closed Principle**.
 
 
-### \[Implemented\] Undo
+### Undo
+The user can undo all previous commands once at a time.
```

Comment:
One* at a time.

Also, does this mean if the user keys in 20 commands, the user can undo all 20 of them?

Reply:
Yes, can type undo until no more previous command